main.py

```diff
@@ -1,12 +1,253 @@
 import requests
+import json
+from resources.frontdesk import show_options, print_inventory, print_accounts
+
+
+def get_path(path):
+    endpoints = {
+        1:'videos/', 
+        2: 'customers/', 
+        3: 'rentals/check-out/', 
+        4: 'rentals/check-in/'
+        }
+
+    return f'https://loucadoura.herokuapp.com/{endpoints[path]}'
+
+
+#=========================HELPERS=========================
+
+
+def valid_video(title, release, inventory):
+
+    if not(title and release and inventory):
+        return False
+
+    return {"title": title, "release_date": release, "total_inventory": inventory}
+
+
+def valid_customer(name, postal_code, phone):
+
+    if not(name and postal_code and phone):
+        return False
+
+    return {"name": name, "postal_code": postal_code, "phone": phone}
+
+
+#=========================EMPLOYEE/STOREROOM=========================
+
+all_videos = {}
+# would be lazy-loading inside its class but no time
+
+
+def get_all_videos():
+
+    if len(all_videos) == 0:
+        path = get_path(1)
+        videos_list = requests.get(path).json()
+
+        for vhs in videos_list:
+            all_videos[vhs["id"]] = vhs
+
+    return all_videos
+
+def get_video(video_id):
+
+    path = get_path(1)
+    video = requests.get(path+f"{video_id}/").json()
```

Comment:
Thanks Jared this is super helpful! I wasn't sure about how I should handle bad requests on this end. I wanted to alter the API code so errors like these would be handled by the original code instead of my local front end 😅 and this would definitely come in handy! thanks a lot :)